Credit Card Validation/src/main/java/CreditCardValidator.java

```diff
@@ -20,32 +26,31 @@ public boolean validateNumbers() {
     if (cardDigitsArray.length == 4) {
       this.creditCardType = getCreditCardType(Integer.parseInt(cardDigitsArray[0]));
       return this.creditCardType != null;
+    } else if (creditCardNumber.length() == 16) {
```

Comment:
Here 16 is magic number. move it to the constants and give name.
Should be something like this:
private static final int CARD_NUMBER_LENGTH = 16;